scripts/llnl_scripts/build_devtools.py

```diff
@@ -50,9 +50,6 @@ def main():
         if not os.path.exists(build_dir):
             os.makedirs(build_dir)
     else:
-        if getpass.getuser() != "atk":
-            print("ERROR: Only shared user 'atk' can install into shared directory. Use -d option.")
-            return 1
```

Comment:
Removing this check expands who can build devtools script to the default shared directory (those with access to the shared directory). 

Are there downsides to removing this?

Reply:
Tag: @white238

Reply:
This guards us against having directories up there that have the wrong permissions and we have to contact LC to remove them

Reply:
> This guards us against having directories up there that have the wrong permissions and we have to contact LC to remove them

Oh, I remember the issue now. I'll revert this change.